contributor_docs/project_wrapups/shantanuKaushik_gsoc_2021.md

```diff
@@ -0,0 +1,57 @@
+  # Adding to p5.js Friendly Error System
+
+#### By Shantanu Kaushik ([@Aloneduckling](https://github.com/Aloneduckling)) | GSoC 2021
+
+### Overview
+Throughout this summer, I worked on p5.js's Friendly Error System (FES in short) under the mentorship of Thales Grilo and Luis Morales-Navarro. The FES is a p5.js feature
+which helps beginners with common errors, transforming error messages into a simple form, and checking for common mistakes. 
```

Comment:
which supports beginners when encountering common errors, checking for common mistakes and providing simple and easy to understand error messages.